src/hotspot/share/classfile/systemDictionary.cpp

```diff
@@ -194,6 +190,38 @@ bool SystemDictionary::is_platform_class_loader(oop class_loader) {
   return (class_loader->klass() == SystemDictionary::jdk_internal_loader_ClassLoaders_PlatformClassLoader_klass());
 }
 
+Handle SystemDictionary::compute_loader_lock_object(Thread* thread, Handle class_loader) {
+  // If class_loader is NULL we synchronize on _system_loader_lock_obj
+  if (class_loader.is_null()) {
+    return Handle(thread, _system_loader_lock_obj.resolve());
+  } else {
+    return class_loader;
+  }
+}
+
+// This method is added to check how often we have to wait to grab loader
+// lock. The results are being recorded in the performance counters defined in
+// ClassLoader::_sync_systemLoaderLockContentionRate and
+// ClassLoader::_sync_nonSystemLoaderLockConteionRate.
```

Comment:
nonSystemLoaderLockContentionRate misspelled.

Reply:
Fixed. thanks for noticing that.